File: BeerViewer/Models/Raw/kcsapi_start2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BeerViewer.Models.Raw
{
	public class kcsapi_start2
	{
		public kcsapi_mst_ship[] api_mst_ship { get; set; }
		public kcsapi_mst_slotitem_equiptype[] api_mst_slotitem_equiptype { get; set; }
		public kcsapi_mst_stype[] api_mst_stype { get; set; }
		public kcsapi_mst_slotitem[] api_mst_slotitem { get; set; }
		public kcsapi_mst_useitem[] api_mst_useitem { get; set; }
		public kcsapi_mst_maparea[] api_mst_maparea { get; set; }
		public kcsapi_mst_mapinfo[] api_mst_mapinfo { get; set; }
		public kcsapi_mission[] api_mst_mission { get; set; }
	}
}

File: WindowFrame/src/KanColle/Raw/Master/kcsapi_start2.ts
// tslint:disable:class-name
import { kcsapi_mst_ship, kcsapi_mst_stype } from "@KC/Raw/Master/kcsapi_mst_ship";
import { kcsapi_mst_slotitem_equiptype, kcsapi_mst_slotitem } from "@KC/Raw/Master/kcsapi_mst_slotitem";
import { kcsapi_mst_useitem } from "@KC/Raw/Master/kcsapi_mst_useitem";
import { kcsapi_mst_maparea, kcsapi_mst_mapinfo } from "@KC/Raw/Master/kcsapi_map";
import { kcsapi_mst_mission } from "@KC/Raw/kcsapi_mission";

export interface kcsapi_start2 {
	/** Ship master information table */
	api_mst_ship: kcsapi_mst_ship[];

	/** Ship type master information table */
	api_mst_stype: kcsapi_mst_stype[];

	/** Equipment master information table */
	api_mst_slotitem: kcsapi_mst_slotitem[];

	/** Equipment type master information table */
	api_mst_slotitem_equiptype: kcsapi_mst_slotitem_equiptype[];

	/** Account item master information table */
	api_mst_useitem: kcsapi_mst_useitem[];

	/**
	 * Map world (Big category) master information table
	 *
	 * `World-Area-Node` for map.
	 * > ex) _**1**_-2-A
	 */
	api_mst_maparea: kcsapi_mst_maparea[];

	/**
	 * Map area (Medium category) master information table
	 *
	 * `World-Area-Node` for map.
	 * > ex) 1-_**2**_-A
	 */
	api_mst_mapinfo: kcsapi_mst_mapinfo[];

	/** Expedition master information table */
	api_mst_mission: kcsapi_mst_mission[];
}
